test/completion-provider.test.ts

```diff
@@ -1,18 +1,18 @@
 import * as assert from 'assert';
 import * as vscode from 'vscode';
 
-async function executeProvider(uri: vscode.Uri, position: vscode.Position): Promise<vscode.CompletionList> {
+export async function executeProvider(uri: vscode.Uri, position: vscode.Position): Promise<vscode.CompletionList> {
```

Comment:
I would prefer (but not super-important) that these files move to a separate file if they are to be reused